### PR TITLE
Added a model downloader, and other tweaks/fixes

### PR DESCRIPTION
**Pull request by pinilpypinilpy**
If enabled, the model downloader will take the provided token, download the model using git lfs, and copy it to the models path.

I replaced condacolab with a regular miniconda install to avoid the kernel restart

I also fixed share_password functionality, as well as a bunch of other fixes.

This is a slightly larger PR than I intended, sorry.

### CODE REVIEW

**Comment by altryne** [comment]
Sorry? haha no worries! I want to .. test it out before.. I wonder if I can somehow do it from the git commit

File: Stable_Diffusion_WebUi_Altryne.ipynb [hunk 1]
```diff
@@ -1,45 +1,10 @@
 {
-  "nbformat": 4,
-  "nbformat_minor": 0,
-  "metadata": {
-    "colab": {
-      "name": "Stable Diffusion WebUi - Altryne",
-      "provenance": [],
-      "collapsed_sections": [
-        "4eodrBxeCgOW",
```
Comment (altryne) [inline Stable_Diffusion_WebUi_Altryne.ipynb:9]: Too bad this got deleted, I think all sections should be collapsed

File: Stable_Diffusion_WebUi_Altryne.ipynb [hunk 2]
```diff
@@ -48,111 +13,102 @@
...
-        "condacolab.install_miniconda()"
-      ],
-      "metadata": {
-        "cellView": "form",
-        "id": "LSCxESSsDGVh"
-      },
-      "execution_count": null,
-      "outputs": []
+        "\n",
+        "#replace condacolab with regular miniconda to avoid the kernel restart\n",
```
Comment (altryne) [inline Stable_Diffusion_WebUi_Altryne.ipynb:64]: this comment prob need to be removed as it only makes sense for us and not users who don't know what either of those are

**Review by altryne — CHANGES_REQUESTED** [review]
Looks really good!  Thank you for this work!

File: Stable_Diffusion_WebUi_Altryne.ipynb (hunk 1 above)
Comment (pinilpypinilpy) [reply Stable_Diffusion_WebUi_Altryne.ipynb:9]: I meant to fix that. Oops.

File: Stable_Diffusion_WebUi_Altryne.ipynb (hunk 2 above)
Comment (pinilpypinilpy) [reply Stable_Diffusion_WebUi_Altryne.ipynb:64]: Fair enough

**Comment by altryne** [comment]
CleanShot 2022-08-30 at 18 07 14@2x
Hmm getting this error

**Comment by altryne** [comment]
But then it works! so weird

**Comment by altryne** [comment]
Could confuse users, but great work! Please collapse the cells by default and I will merge 🙏

**Comment by pinilpypinilpy** [comment]
I think the reason for the git error is because the model size is around 4gb, which is the maximum size that can be copied directly to a fat32-formatted disk? Not an issue here though.

**Comment by pinilpypinilpy** [comment]
I noticed that the last cell isn't collapsed by default, is that intentional?

**Comment by pinilpypinilpy** [comment]
The reason that the metadata was missing to begin with is because VS code strips notebooks of metadata when I save them, for some reason.

**Comment by pinilpypinilpy** [comment]
@altryne I think I've figured it out. VS code moved the metadata to the bottom, and I re-added it to the top. Now there's two of the same metadata sections.

**Pull request merged**